include/tango/client/device/device_command_info.hxx

```diff
@@ -0,0 +1,82 @@
+//
```

Comment:
Project code style has been added to the master branch, see ea387e6. Also a new wiki page has been created: [Code style guidelines](https://github.com/tango-controls/cppTango/wiki/Code-style-guidelines)